recipes/configs/qwen2_5/0_5B_lora.yaml

```diff
@@ -20,11 +20,11 @@
 # Model Arguments
 model:
   _component_: torchtune.models.qwen2_5.lora_qwen2_5_0_5b
-  lora_attn_modules: ['q_proj', 'v_proj']
-  apply_lora_to_mlp: False
+  lora_attn_modules: ['q_proj', 'v_proj', 'output_proj']
+  apply_lora_to_mlp: True
   apply_lora_to_output: False
```

Comment:
I know it's not from your PR but I thought Qwen 2.5 has tied word embeddings for smaller model sizes. In that case we should not even be exposing `apply_lora_to_output` here?